ui/page/odyseeMembership/view.jsx

```diff
@@ -0,0 +1,462 @@
+// @flow
```

Comment:
This component contains template literals, concatenated strings, and no i18n macro.
If this component will forever be in English only, then it is fine -- otherwise, please prepare for i18n.